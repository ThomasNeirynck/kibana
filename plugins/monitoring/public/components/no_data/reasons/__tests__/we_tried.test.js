import React from 'react';
import { render } from 'enzyme';
import { WeTried } from '../';

describe('WeTried', () => {
  test('should render "we tried" message', () => {
    const component = render(<WeTried />);
    expect(component).toMatchSnapshot();
  });
});
