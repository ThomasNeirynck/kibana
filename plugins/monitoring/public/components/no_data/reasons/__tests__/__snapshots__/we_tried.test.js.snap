// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WeTried should render "we tried" message 1`] = `
Array [
  <h3
    class="euiTitle euiTitle--small"
  >
    We tried looking for the reason that monitoring data is unavailable.
  </h3>,
  <p>
    No Monitoring data could be found for the selected time period, but we could not find the cluster setting that makes the data unavailable.
  </p>,
  <p>
    There may be data available for a different time period than we have selected. Try adjusting the time filter controls to a time range where the Monitoring data is expected.
  </p>,
  <p>
    We are refreshing the search for data in the background. If cluster data is found, we well redirect to the cluster overview page.
  </p>,
]
`;
